test/addons/buffer-free-callback/test.js

```diff
@@ -16,20 +16,26 @@ function check(size, alignment, offset) {
   global.gc();
 }
 
+// NOTE : It is discouraged to add more check()
+// test. If it is absolutely necessary to do so,
+// make sure there's no duplicated pair of
+// (alignment, offset).
+// Info: https://github.com/nodejs/node/issues/31061
```

Comment:
I don’t think we need to discourage adding any more checks, just the ones with duplicated alignment and offset pairings.

Reply:
Maybe this is all that's needed for the comment?:

```javascript
// If adding more check() test cases, be sure to not duplicate alignment/offset.
// Refs: https://github.com/nodejs/node/issues/31061#issuecomment-568612283
```